xbdd-web/__test__/containers/ScenarioListViewContainer-test.js

```diff
@@ -0,0 +1,8 @@
+// ScenarioListViewContainer test goes here
```

Comment:
Probably don't need this file as part of your PR. I have already made changes to it. :)